src/main/java/training/commands/Roller/diskOutCommand.java

```diff
@@ -0,0 +1,11 @@
+package training.commands.Roller;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+import static training.commands.Roller.Constants.ROLLER_ROLLING_TIME_SEC;
+
+public class diskOutCommand extends SequentialCommandGroup {
```

Comment:
```suggestion
public class NoteOutCommand extends SequentialCommandGroup {
```
change in all files, big letter at the begining of the class name